fe/fe-core/src/main/java/org/apache/doris/job/scheduler/JobScheduler.java

```diff
@@ -124,6 +124,10 @@ public void scheduleOneJob(T job) throws JobException {
                 schedulerInstantJob(job, TaskType.SCHEDULED, null);
             }
         }
+        if (job.getJobConfig().isImmediate() && JobExecuteType.ONE_TIME.equals(job.getJobConfig().getExecuteType())) {
```

Comment:
why JobExecuteType == ONE_TIME ,can not setLatestSchedulerTimeMs(System.currentTimeMillis())?

Reply:
This variable is not persistent, so for a one-time, immediately scheduled task, there's no need to set it.